src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -1285,6 +1285,70 @@ featurizers:
     # we introduce the tabular data type itself).
     status: External
 
+  # ----------------------------------------------------------------------
+  - name: LagLeadOperatorFeaturizer
+    estimator_name: LagLeadOperatorEstimator
+    release_version: 0.4.0
+    num_output_columns: 1
+    description: |-
+      Computes lags and leads for values. Works for general time series data sets.
+      For sparse time series data sets, forward fill imputer will be applied.
+
+      C++-style pseudo signature:
+        template <typename T> double execute(std::tuple<std::chrono::system_clock::time_point, T, uint8> const &value);
+
+      Examples:
+          Assuming we have max horizon = 2 and we want to lag 1 and 2
+          +---------------+-------+------------------+-------------------+-------------------+
+          | time          | target| horizon_origin   | target_lag_1      | target_lag_2      |
+          +===============+=======+==================+===================+===================+
+          | 2017-01-01    | 8     | 2016-11-01       | NAN               | NAN               |
+          +---------------+-------+------------------+-------------------+-------------------+
+          | 2017-01-01    | 8     | 2016-12-01       | NAN               | NAN               |
+          +---------------+-------+------------------+-------------------+-------------------+
+          | 2017-02-01    | 9     | 2016-12-01       | NAN               | NAN               |
+          +---------------+-------+------------------+-------------------+-------------------+
+          | 2017-02-01    | 9     | 2017-01-01       | 8                 | NAN               |
+          +---------------+-------+------------------+-------------------+-------------------+
+          | 2017-03-01    | 10    | 2017-01-01       | 8                 | NAN               |
+          +---------------+-------+------------------+-------------------+-------------------+
+          | 2017-03-01    | 10    | 2017-02-01       | 9                 | 8                 |
+          +---------------+-------+------------------+-------------------+-------------------+
+          | 2017-04-01    | 11    | 2017-02-01       | 10                | 8                 |
+          +---------------+-------+------------------+-------------------+-------------------+
+          | 2017-04-01    | 11    | 2017-03-01       | 11                | 10                |
+          +---------------+-------+------------------+-------------------+-------------------+

```

Comment:
I think the dates in the horizon column are messed up. Seems like for the first rows you are putting months in the day spot.

Reply:
As discussed offline put the date format you are using.